examples/README.md

```diff
@@ -11,6 +11,7 @@ Distro:
 - [`debian.yaml`](./debian.yaml): ⭐Debian GNU/Linux
 - [`fedora.yaml`](./fedora.yaml): ⭐Fedora
 - [`opensuse.yaml`](./opensuse.yaml): ⭐openSUSE Leap
+- [`oracle.yaml`](./oracle.yaml): Oracle Linux
```

Comment:
Maybe, the file name should be “oraclelinux.yaml”?
“oracle” means the database product to me.
(For the same reason the ArchLinux yaml is named “archlinux.yaml”, not “arch.yaml”)

Reply:
Yeah fair point. Moved the file and updated readme.